pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -227,42 +227,25 @@ in {
     latestCompatibleLinuxPackages = linuxPackages_6_1;
 
     # this package should point to the latest release.
-    version = "2.1.9";
+    version = "2.1.10";
 
-    sha256 = "RT2ijcXhdw5rbz1niDjrqg6G/uOjyrJiTlS4qijiWqc=";
+    sha256 = "sha256-ZREe6r7Z8JPj6dEmJesU1hVFwegx0J5CYeZ2b4draKY=";
   };
 
   zfsUnstable = common {
     # check the release notes for compatible kernels
-    # NOTE:
-    #   zfs-2.1.9<=x<=2.1.10 is broken with aarch64-linux-6.2
-    #   for future releases, please delete this condition.
```

Comment:
Are we sure this issue has been fixed? The commit adding this comment doesn't link to an issue. Does anyone know what it was? @PedroHLC maybe?

Reply:
It was an issue due to kernel neon start symbols.

Reply:
Hm yea I see that now. And I'm still getting that build error with this PR and linux 6.2, on both `zfs` and `zfsUnstable`

Reply:
```
ERROR: modpost: GPL-incompatible module zcommon.ko uses GPL-only symbol 'kernel_neon_begin'
ERROR: modpost: GPL-incompatible module zcommon.ko uses GPL-only symbol 'kernel_neon_end'
ERROR: modpost: GPL-incompatible module zfs.ko uses GPL-only symbol 'kernel_neon_end'
ERROR: modpost: GPL-incompatible module zfs.ko uses GPL-only symbol 'kernel_neon_begin'
```

Reply:
The fix is to revert a certain commit, which has no other effect than breaking ZFS. :)